### DISCUSSION TITLE
.geogram always stores attributes even if the default ioflag is used (the default turns off attribute storage)

### DISCUSSION THREAD

**Discussion by skandm** [BrunoLevy/geogram#86] [Q&A]
I just want to store the vertices, cells, etc. and not store any attributes attached to it. Is there a way to do it? Seems to me that it is not possible and is a bug. Thoughts?

**Comment by BrunoLevy** [comment]
Hi there,
I'm unsure I understood what you mean (if you do not declare any attribute, it will not allocate any storage for the attributes).

**Reply by skandm** [reply]
let's say i have a vertex attribute defined "my custom data". When saving the .geogram format I do not want to write out the "my custom data". The way to do it will be to provide the MeshIOFlags. The default constructor of MeshIOFlags set the attributes to MESH_NO_ATTRIBUTES. Hence, using the default MeshIOFlag should not write the attribute out. But in the save() function for .geogram extension it does not check for the attribute IOFlag and writes out the "my custom data" regardless of what the ioflag is set to be.

**Reply by BrunoLevy** [reply]
Thanks, it is probably something I forgot to implement, I'll take a look at that.
